webrtc.html

```diff
@@ -5139,7 +5161,8 @@ <h2>RTP Media API</h2>
       created. <code><a>RTCRtpSender</a></code>s and
       <code><a>RTCRtpReceiver</a></code>s are always created at the same time
       as an <code><a>RTCRtpTransceiver</a></code>, which they will remain
-      attached to for their lifetime.
+      attached until it has been <a data-lt="stop the RTCRtpTransceiver">
+      stopped</a> and the fact that it has been stopped has been negotiated.
```

Comment:
It is the transceiver that is no longer attached, senders and receivers are always attached to the transceiver.